doc/nrf/releases/release-notes-2.2.0.rst

```diff
@@ -0,0 +1,116 @@
+.. _ncs_release_notes_220:
+
+|NCS| v2.2.0 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+** FIX THIS ? **
```

Comment:
Is this file in place somewhere now so that you can add the needed content there?

Reply:
No I have created the file, because somehow release notes are crated at the and when we forget what has been done, so  I left note here that I have not touched the stuff here.

Reply:
You can add an entry in this file https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/releases/release-notes-changelog.rst#L29